src/typeDefs/schema.graphql

```diff
@@ -7,16 +7,12 @@ type Contact {
 
 type Facility {
   id: ID!
-  names: [FacilityName]!
```

Comment:
I simplified the facility names on the DB end and am adjusting the GraphQL to match.

My reasoning: Most medical websites in Japan have both a Japanese and an English or Romaji name. I haven't seen clinics using, say, Vietnamese names, etc., and I thought an overly complex model would be harder for MVP. Might be my own bias, and checking with Russ 🤔